hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/parser/record/ethereum/Eip2930EthereumTransactionParser.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2022-2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.mirror.importer.parser.record.ethereum;
+
+import com.esaulpaugh.headlong.rlp.RLPDecoder;
+import com.hedera.mirror.common.domain.transaction.EthereumTransaction;
+import com.hedera.mirror.importer.exception.InvalidEthereumBytesException;
+import jakarta.inject.Named;
+
+@Named
+public class Eip2930EthereumTransactionParser implements EthereumTransactionParser {
+    public static final int EIP2930_TYPE_BYTE = 1;
+    public static final String EIP2930_TYPE_HEX = "0x01";
+    private static final String TRANSACTION_TYPE_NAME = "EIP2930";
+    private static final int EIP2930_TYPE_RLP_ITEM_COUNT = 11;
+
+    @Override
+    public EthereumTransaction decode(byte[] transactionBytes) {
+        var decoder = RLPDecoder.RLP_STRICT.sequenceIterator(transactionBytes);
+        var legacyRlpItem = decoder.next();
+        var legacyRlpItemByte = legacyRlpItem.asByte();
+        if (legacyRlpItemByte != EIP2930_TYPE_BYTE) {
```

Comment:
Why is this called a legacyRlpItem?

```suggestion
        var type = decoder.next();
        var typeByte = type.asByte();
        if (typeByte != EIP2930_TYPE_BYTE) {
```